app/views/employees/check.html.erb

```diff
@@ -1,3 +1,58 @@
-<p style="color: green"><%= notice %></p>
+<div class = "container">
+
+  <header>
+      <div class="logo">
+        <h1>FOO CORP</h1>
+      </div>
+  </header>
+
+  <main>
+
+    <div class="check">
+
+      <div class="numeric-pad">
+
+        <div class="screen">
+          <p>000000</p>
+        </div>
+
+        <div class="numbers-button">
+          <input type="button" value="1">
+          <input type="button" value="2">
+          <input type="button" value="3">
+          <input type="button" value="4">
+          <input type="button" value="5">
+          <input type="button" value="6">
+          <input type="button" value="7">
+          <input type="button" value="8">
+          <input type="button" value="9">
+          <input type="button" value="0">
+          <input type="button" value="◄" onclick="test()">
+        </div>
+      </div>
+
+      <div class="checks-buttons">
+        <button class="primary-button">IN</button>
+        <button class="primary-button">OUT</button>
+      </div>
+
+    </div>
+
+    <div class="right-side">
+
+      <div class="clock">
+      <p>08:10 AM</p>
+      <p>Sunday</p>
+      <p>07 / Mar / 2022</p> 
+      </div>
+
+      <button class="primary-button">Admin</button>
+
+    </div>
+    
+  </main>
+
+  
+
```

Comment:
Remove unnecessary spaces in this file.

Reply:
Thanks for the review, we will make the changes.